test/e2e_node/e2e_service.go

```diff
@@ -208,6 +209,24 @@ func (es *e2eService) startServer(cmd *healthCheckCommand) error {
 	return fmt.Errorf("Timeout waiting for service %s", cmd)
 }
 
+// Special log files that need to be collected for additional debugging.
+var logFiles = []string{
+	"/var/log/kern.log",
+	"/var/log/messages",
+	"/var/log/docker.log",
+}
+
+// Add hard links to log files that are of interest.
+// Since we scp files from the remote directory, hardlinks will be treated as normal files and file contents will be copied over.
+func linkLogFiles() {
+	for _, file := range logFiles {
+		targetLink := path.Join(*reportDir, path.Base(file))
+		if err := os.Symlink(file, targetLink); err != nil {
```

Comment:
fwiw, this will fail if the logs are on a different partition because you can't link across partitions.  That may be fine for now.